### ISSUE TITLE
warning in the newest version (0.27.1) of css-loader

### ISSUE THREAD

**Issue by borisirota** [webpack-contrib/source-map-loader#34]
`css-loader` was updated and [this comment](https://github.com/webpack-contrib/css-loader/blob/master/lib/convert-source-map.js#L11) was added.

I think that this is the reason for the next warning ->
```
./~/css-loader/lib/convert-source-map.js
Cannot find SourceMap 'foo.js.map': Error: Can't resolve './foo.js.map' in 'PATH_TO_PROJECT/node_modules/css-loader/lib'
 @ ./~/css-loader/lib/css-base.js 56:24-55
 @ ./~/css-loader?{"autoprefixer":false}!./~/postcss-loader!./~/less-loader/lib/loader.js!./src/web/global.less
 @ ./src/web/global.less
 @ ./src/web/components/App/index.jsx
 @ ./src/web/app.jsx
 @ multi (webpack)-dev-server/client?http://0.0.0.0:3000 webpack/hot/dev-server ./src/web/app
```

**Comment by michael-ciniawsky** [comment]
@borisirota Wrong repo 🙃 . Could you please reopen in `css-loader`  the module here isn't used by `css-loader`, since this seems to be some kind of workaround in `css-loader`, it's likely you're right 😛

**Comment by michael-ciniawsky** [comment]
Tracking Issue is here https://github.com/webpack/webpack/issues/4458

**Comment by borisirota** [comment]
@michael-ciniawsky Thanks for the quick reply and sorry for not being 100% clear about my usage. I'm using both `source-map-loader` and `css-loader` and since the last update of `css-loader` I see this warning.

Does `source-map-loader` suppose to intercept [this example comment](https://github.com/webpack-contrib/css-loader/blob/master/lib/convert-source-map.js#L11) as trigger to find the source map file ?

I'm not in the implementation details so not quite understand how the tracking issue is related or suppose to remove this warning.

Thanks !

**Comment by michael-ciniawsky** [comment]
@borisirota kk I must check and any logs etc are appreciated, if I rememver right, there was a similiar issue before, but I tought it was fixed 😛 . In any case I remove the comment in `css-loader` to fix asap and if `source-map-loader` has a flanky RegExp check somewhere we must debug it first :D

**Comment by michael-ciniawsky** [comment]
This was the last change [#31](https://github.com/webpack-contrib/source-map-loader/pull/31/files#diff-168726dbe96b3ce427e7fedce31bb0bcR12)

**Comment by michael-ciniawsky** [comment]
Fixed by [#452](https://github.com/webpack-contrib/css-loader/pull/452)

**Comment by borisirota** [comment]
@michael-ciniawsky great thanks !
